### PR TITLE
Change Contract Interface Keyword

### PR DESCRIPTION
**Pull request by fubuloubu**
Closes: #1825

### Description for the changelog
Changed Contract Interface Keyword from "contract" to "interface"

### Cute Animal Picture
ugh

### CODE REVIEW

File: vyper/ast/pre_parser.py [hunk 1]
```diff
@@ -114,13 +114,22 @@ def pre_parse(code: str) -> Tuple[ClassTypes, str]:
...
+                    "Perhaps you meant `interface` or `struct`?",
                     code,
                     start[0],
                     start[1],
                 )
 
-            # Make note of contract or struct name along with the type keyword
+            if typ == NAME and string == "contract" and start[1] == 0:
+                raise SyntaxException(
+                    "The `contract` keyword has been deprecated. Please use `interface`?",
```
Comment (iamdefinitelyahuman) [inline vyper/ast/pre_parser.py:126]: Is that question mark intentional?

Comment (fubuloubu) [reply vyper/ast/pre_parser.py:126]: No lol

Comment (fubuloubu) [reply vyper/ast/pre_parser.py:126]: I think I was trying to say "did you mean to use interface?"

**Comment by fubuloubu** [comment]
TODO: docs?

**Pull request merged**